src/calendar/stories/RangeCalendar.stories.tsx

```diff
@@ -71,24 +71,22 @@ export const Default = Base.bind({});
 
 export const DefaultValue = Base.bind({});
 DefaultValue.args = {
-  start: new Date(),
-  end: addWeeks(new Date(), 1),
   defaultStart: new Date(),
   defaultEnd: addWeeks(new Date(), 1),
 };
 
 export const MinMaxDefaultDate = Base.bind({});
 MinMaxDefaultDate.args = {
-  start: new Date(),
-  end: addDays(new Date(), 1),
+  defaultStart: new Date(),
+  defaultEnd: addDays(new Date(), 1),
   minValue: subWeeks(new Date(), 1),
   maxValue: addWeeks(new Date(), 1),
 };
 
 export const Options = Base.bind({});
 Options.args = {
-  start: new Date(),
-  end: addWeeks(new Date(), 1),
+  defaultStart: new Date(),
+  defaultEnd: addWeeks(new Date(), 1),
   minValue: null,
   maxValue: null,
   isDisabled: false,
```

Comment:
If this is intentional then it's fine but using default values means we cannot change the values from the storybook control addon

Reply:
Oh, right right !! I did not know this !!

I will revert back.